src/Silex/Application.php

```diff
@@ -246,10 +245,10 @@ public function before($callback, $priority = 0)
     {
         $app = $this;
 
-        $this->on(KernelEvents::REQUEST, function (GetResponseEvent $event) use ($callback, $app) {
-            if (!$event->isMasterRequest()) {
-                return;
-            }
+        $this->on(KernelEvents::REQUEST, function (RequestEvent $event) use ($callback, $app) {
```

Comment:
https://github.com/symfony/http-kernel/blob/v4.4.50/Event/GetResponseEvent.php

```
/**
 * @deprecated since Symfony 4.3, use RequestEvent instead
 */
```